package-builder/flox-build.mk

```
@@ -89,8 +98,16 @@ TMPDIR ?= /tmp
 # Use the wildcard operator to identify builds in the provided $FLOX_ENV.
 BUILDS := $(wildcard $(FLOX_ENV)/package-builds.d/*)
 
+# TODO NIX_EXPRESSION_DIR may be absent
+ifeq (,$(NIX_EXPRESSION_DIR))
+  $(error NIX_EXPRESSION_DIR not defined)
+endif
+NIX_EXPRESSION_BUILDS := $(shell $(_nix) eval --argstr nixpkgs-url $(BUILDTIME_NIXPKGS_URL) --argstr system $(NIX_SYSTEM) --argstr pkgs-dir $(NIX_EXPRESSION_DIR) --file $(__nef) reflect.targets --raw)
+
 # Quick sanity check; if no BUILDS then what are we doing?
-$(if $(BUILDS),,$(error no packages found in $(FLOX_ENV)/package-builds.d))
+#
+# We might eb building nef, so, keep on
+#$(if $(BUILDS),,$(error no packages found in $(FLOX_ENV)/package-builds.d))
```

This ought to do it:
```suggestion
$(if $(BUILDS),,\
  $(if $(NIX_EXPRESSION_BUILDS),,\
    $(error no package or Nix expression builds found in $(FLOX_ENV))))
```

---

I believe the error message will be wrong for nef since the expressions won't be in $FLOX_ENV